geomesa-core/src/test/scala/org/locationtech/geomesa/core/iterators/TemporalDensityIteratorTest.scala

```diff
@@ -117,7 +128,7 @@ class TemporalDensityIteratorTest extends Specification {
       (iter must not).beNull
 
       iter.length should be lessThan 150
-      iter.length should be equalTo 1
+      iter.length mustEqual 1
```

Comment:
This is fine (it's my preference) -- but "should be equalTo" and "mustEqual" are functionally equivalent.